specification/introduction/introduction.tex

```diff
@@ -3,59 +3,81 @@ \chapter{Introduction}\label{sec:introduction}
 UAVCAN is a lightweight protocol designed to provide a highly reliable communication method for
 aerospace and robotic applications via robust vehicle bus networks.
 
-This is a non-normative section covering the basic concepts that govern development and maintenance of the specification.
+This is a non-normative section covering the basic concepts that govern development and maintenance of
+the specification.
 
 \section{Design principles}
 
-UAVCAN is designed to adhere to the following set of basic principles.
-
 \begin{description}
     \item[Democratic network] --- There will be no master node.
     All nodes in the network will have the same communication rights; there must be no single point of failure.
 
-    \item[Nodes can exchange long payloads] --- Nodes must be provided with a simple way of exchanging large
-    data structures that cannot fit into a single transport frame (such as GNSS solutions, 3D vectors, etc.).
-    UAVCAN should perform automatic transfer decomposition and reassembly at the protocol level,
+    \item[Facilitation of functional safety] --- A system designer relying on UAVCAN will have the necessary
+    guarantees and tools at their disposal to analyze the system and ensure its correct behavior.
+
+    \item[High-level communication abstractions] --- The protocol will support publish/subscribe and remote procedure
+    call communication semantics with statically defined and statically verified data types (schema).
+    The data types used for communication will be defined in a clear, platform-agnostic way
+    that can be easily understood by machines, including humans.
+
+    \item[Facilitation of cross-vendor interoperability] --- UAVCAN will be a common foundation that
+    different vendors can build upon to ensure interoperability of their equipment.
+    UAVCAN will provide a generic set of standard application-agnostic communication data types.
+
+    \item[Well-defined generic high-level functions] --- UAVCAN will define standard services
+    and messages for common high-level functions, such as network discovery, node configuration,
+    node software update, node status monitoring\footnote{Which naturally grows into a vehicle-wide
+    health monitoring solution.}, network-wide time synchronization, plug-and-play node support, etc.
+
+    \item[Atomic data abstractions] --- Nodes must be provided with a simple way of exchanging large
+    data structures that exceed the capacity of a single transport frame\footnote{A \emph{transport frame} is
+    an atomic transmission unit defined by the underlying transport protocol. For example, a CAN frame.}.
+    UAVCAN should perform automatic data decomposition and reassembly at the protocol level,
     hiding the related complexity from the application.
 
+    \item[High throughput, low latency, determinism] --- UAVCAN must be suitable for use in
```

Comment:
> This is a bit "Yes we want them alls"

The listed properties are not mutually exclusive so I don't think there is a problem. Do you have a better wording in mind? We shouldn't mention CAN specifically because we are now officially transport-agnostic.

Reply:
It's hard without mentioning CAN. What about: "Uavcan adds very little overhead to the transport layer, giving it high throughput while retaining low latency and suitability for hard real time usage."

Reply:
:+1: